internal/httputil/client.go

```diff
@@ -14,17 +14,29 @@ import (
 
 	"go.opencensus.io/plugin/ochttp"
 
+	"github.com/pomerium/pomerium/internal/telemetry/metrics"
 	"github.com/pomerium/pomerium/internal/telemetry/requestid"
+	"github.com/pomerium/pomerium/internal/tripper"
 )
 
 // ErrTokenRevoked signifies a token revokation or expiration error
 var ErrTokenRevoked = errors.New("token expired or revoked")
 
+type httpClient struct {
+	*http.Client
+	requestIDTripper http.RoundTripper
+}
+
+func (c *httpClient) Do(req *http.Request) (*http.Response, error) {
+	tripperChain := tripper.NewChain(metrics.HTTPMetricsRoundTripper("idp_http_client", req.RequestURI))
```

Comment:
```suggestion
	tripperChain := tripper.NewChain(metrics.HTTPMetricsRoundTripper("idp_http_client", req.Host))
```
RequestURI is only good on the server side

Reply:
Fixed